drams/hello_clojure.clj

```diff
@@ -121,7 +121,7 @@
   map             ; symbol
   +               ; symbol - most punctuation allowed
   clojure.core/+  ; namespaced symbol
-  nil             ; null value
+  nil             ; nil value
```

Comment:
That is debatable. 😄 
https://clojure.org/guides/learn/syntax#_symbols_and_idents

```suggestion
  nil             ; null/nil value (named in the LISP tradition)
```